test/JDBC/expected/BABEL-5059-vu-verify.out

```diff
@@ -0,0 +1,1248 @@
+-- Procedure
```

Comment:
Is this new file part of your changes ?

Reply:
this file was there in 4x but not in 3x, so it get cherry picked here, removing it here as it is not necessary